.github/workflows/tag_release.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       env:
         CHANGELOG_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Run GoReleaser
-      uses: goreleaser/goreleaser-action@108bc4d
+      uses: goreleaser/goreleaser-action@v2
       with:
-        version: 'v0.135.0'
+        version: latest
```

Comment:
There was a preference from team-sec from our original security review of this project that we pin to specfic versions instead of using `latest`.

Reply:
@phamann I'm not sure where this 'version' came from. Looking at the goreleaser repo it didn't have this version as either a tag or a branch. There was a [`0.1.0`](https://github.com/goreleaser/goreleaser-action/releases/tag/v0.1.0) but not `0.135.0` 🤔

Reply:
Although now looking at the code. I'm not use what `with` `version` actually means?

Reply:
Ah ok so reading https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith this would suggest that `version` is being passed into the goreleaser, so there appears to be _two_ distinct 'versions' at play: 

1. the version we pass in (e.g. `latest` or `v0.135.0`)
2. the release version of goreleaser (e.g. `@v2`)

Reply:
Yup so the version `v0.135.0` _should_ have aligned with some kind of tag I think based on the goreleaser docs:

https://github.com/goreleaser/goreleaser-action#inputs

> Can be a fixed version like v0.117.0 or a max satisfying semver one like ~> 0.132. In this case this will return v0.132.1.

But I'm still not sure where we got that original version from as it doesn't show up anywhere 🤔